static/css/index.css

```diff
@@ -552,6 +598,11 @@ input[type="submit"]:focus {
   background-color: var(--button-hover-color);
 }
 
+#newsletter-form input[type="submit"] {
+  font-weight: 600;
```

Comment:
```suggestion
  font-weight: var(--font-weight-medium);
```

Reply:
I think that the semi-bold font weight stands out to much so medium is better in this case.

Reply:
Okay will surely do the same

Reply:
Done @colinkiama

Reply:
Done @colinkiama